templates/js/dashboard-tab/env/.env.dev

```diff
@@ -11,5 +11,4 @@ RESOURCE_SUFFIX=
 
 # Generated during provision, you can also add your own variables.
 TEAMS_APP_ID=
-TAB_AZURE_STORAGE_RESOURCE_ID=
```

Comment:
Maybe add AZURE_STATIC_WEB_APPS_RESOURCE_ID

Reply:
Changed to AZURE_STATIC_WEB_APPS_RESOURCE_ID in all places.

Reply:
@hund030  This a real case of action owner is working on updating all (Tab) templates with new action. Maybe we can take this as an exercise and ask @Siglud about his experience of using Template Builder tools.